### PR TITLE
Block entity constants

### PR DESCRIPTION
**Pull request by IotaBread**

### CODE REVIEW

File: mappings/net/minecraft/block/entity/SkullBlockEntity.mapping [hunk 1]
```diff
@@ -16,11 +16,16 @@ CLASS net/minecraft/unmapped/C_vrhaavsg net/minecraft/block/entity/SkullBlockEnt
 	METHOD m_btvvujwr (Ljava/util/function/Consumer;Lcom/mojang/authlib/GameProfile;)V
 		ARG 1 profile
 	METHOD m_fanntxcn getOwner ()Lcom/mojang/authlib/GameProfile;
+	METHOD m_flpykord setServices (Lnet/minecraft/unmapped/C_jyqwslie;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Ljava/util/concurrent/Executor;)V
+		ARG 0 userCache
+		ARG 1 serrsionService
```
Comment (triphora) [inline mappings/net/minecraft/block/entity/SkullBlockEntity.mapping]:
```suggestion
		ARG 1 sessionService
```

**Review by triphora — CHANGES_REQUESTED** [review]

**Review by triphora — APPROVED** [review]

File: mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping [hunk 2]
```diff
@@ -1,13 +1,23 @@
 CLASS net/minecraft/unmapped/C_dlxbwxyf net/minecraft/block/entity/AbstractFurnaceBlockEntity
 	FIELD f_apovtvbw SIDE_SLOTS [I
+	FIELD f_bitqkham BURN_TIME_STANDARD I
+	FIELD f_cjviljdx DATA_FUEL_TIME I
+	FIELD f_debmtawn RESULT_SLOT I
 	FIELD f_dswenqhr cookTimeTotal I
+	FIELD f_erasrdjo FUEL_SLOT I
 	FIELD f_fwsjbtjw TOP_SLOTS [I
+	FIELD f_hpqapnfb BURN_COOL_SPEED I
```
Comment (OroArmor) [inline mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping]: This is vague and seems like a type (I know its not) Could it be clarified?

**Review by LambdAurora — APPROVED** [review]

File: mappings/net/minecraft/block/entity/ConduitBlockEntity.mapping [hunk 3]
```diff
@@ -1,12 +1,18 @@
 CLASS net/minecraft/unmapped/C_gjnfxnhb net/minecraft/block/entity/ConduitBlockEntity
 	FIELD f_ceaxiukt activatingBlocks Ljava/util/List;
+	FIELD f_cjccavbs EFFECT_DURATION I
 	FIELD f_cjxkooge ticks I
+	FIELD f_emfbfiqt KILL_RANGE I
 	FIELD f_epsxshbm eyeOpen Z
+	FIELD f_jigkkgnf MIN_KILL_SIZE I
```
Comment (OroArmor) [inline mappings/net/minecraft/block/entity/ConduitBlockEntity.mapping]: I would use `DAMAGE` not `KILL` since the conduit applies damage over time

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping [hunk 4]
```diff
@@ -1,13 +1,23 @@
 CLASS net/minecraft/unmapped/C_dlxbwxyf net/minecraft/block/entity/AbstractFurnaceBlockEntity
 	FIELD f_apovtvbw SIDE_SLOTS [I
+	FIELD f_bitqkham BURN_TIME_STANDARD I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping]:
```suggestion
	FIELD f_bitqkham STANDARD_BURN_TIME I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping (hunk 2 above)
Comment (supersaiyansubtlety) [reply mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping]: I think "decay" would be appropriate here, perhaps 
```suggestion
	FIELD f_hpqapnfb BURN_DECAY I
```
or
```suggestion
	FIELD f_hpqapnfb BURN_DECAY_RATE I
```
?

File: mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping [hunk 5]
```diff
@@ -1,13 +1,23 @@
...
+	FIELD f_cjviljdx DATA_FUEL_TIME I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping]:
```suggestion
	FIELD f_cjviljdx FUEL_TIME_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping [hunk 6]
```diff
@@ -1,13 +1,23 @@
...
+	FIELD f_debmtawn RESULT_SLOT I
 	FIELD f_dswenqhr cookTimeTotal I
+	FIELD f_erasrdjo FUEL_SLOT I
 	FIELD f_fwsjbtjw TOP_SLOTS [I
+	FIELD f_hpqapnfb BURN_COOL_SPEED I
 	FIELD f_kekgcosu propertyDelegate Lnet/minecraft/unmapped/C_fwwsyhuv;
 	FIELD f_mzaccpku BOTTOM_SLOTS [I
+	FIELD f_nqwyvoxy DATA_COOK_TIME_TOTAL I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping]:
```suggestion
	FIELD f_nqwyvoxy TOTAL_COOK_TIME_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping [hunk 7]
```diff
@@ -1,13 +1,23 @@
...
 	FIELD f_ridjaugn recipeType Lnet/minecraft/unmapped/C_rhnqznys;
+	FIELD f_ruznenwn INPUT_SLOT I
 	FIELD f_usgwvvfo cookTime I
+	FIELD f_uunwympm DATA_BURN_TIME I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping]:
```suggestion
	FIELD f_uunwympm BURN_TIME_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping [hunk 8]
```diff
@@ -1,13 +1,23 @@
...
 	FIELD f_vmpcjcui recipesUsed Lit/unimi/dsi/fastutil/objects/Object2IntOpenHashMap;
+	FIELD f_vnrsgghy DATA_COOK_TIME I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping]:
```suggestion
	FIELD f_vnrsgghy COOK_TIME_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping [hunk 9]
```diff
@@ -4,9 +4,16 @@ CLASS net/minecraft/unmapped/C_wwadquuj net/minecraft/block/entity/BeaconBlockEn
 	FIELD f_fvmnfejs propertyDelegate Lnet/minecraft/unmapped/C_fwwsyhuv;
 	FIELD f_galoemma customName Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_gdxyfopa EFFECTS Ljava/util/Set;
+	FIELD f_iespktor DATA_VALUES I
+	FIELD f_mhasxwwh DATA_PRIMARY I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping]:
```suggestion
	FIELD f_mhasxwwh PRIMARY_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping [hunk 10]
```diff
@@ -4,9 +4,16 @@ CLASS net/minecraft/unmapped/C_wwadquuj net/minecraft/block/entity/BeaconBlockEn
...
+	FIELD f_noizsupm BLOCKS_CHECK_PER_TICK I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping]:
```suggestion
	FIELD f_noizsupm BLOCKS_CHECKED_PER_TICK I
```
or
```suggestion
	FIELD f_noizsupm BLOCK_CHECKS_PER_TICK I
```

File: mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping [hunk 11]
```diff
@@ -4,9 +4,16 @@ CLASS net/minecraft/unmapped/C_wwadquuj net/minecraft/block/entity/BeaconBlockEn
...
 	FIELD f_olqmkzcd minY I
+	FIELD f_pczjddup DATA_SECONDARY I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping]:
```suggestion
	FIELD f_pczjddup SECONDARY_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping [hunk 12]
```diff
@@ -4,9 +4,16 @@ CLASS net/minecraft/unmapped/C_wwadquuj net/minecraft/block/entity/BeaconBlockEn
...
+	FIELD f_qqcnhnwr DATA_LEVEL I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping]:
```suggestion
	FIELD f_qqcnhnwr LEVEL_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping [hunk 13]
```diff
@@ -4,9 +4,16 @@ CLASS net/minecraft/unmapped/C_wwadquuj net/minecraft/block/entity/BeaconBlockEn
...
 	FIELD f_qtfjebrg primary Lnet/minecraft/unmapped/C_jaqasomh;
 	FIELD f_rkpllpyn EFFECTS_BY_LEVEL [[Lnet/minecraft/unmapped/C_jaqasomh;
+	FIELD f_ryyxokfs MAX_LEVEL I
+	FIELD f_ugyhjati checkingBeamSegments Ljava/util/List;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping]: This sounds like a name for a boolean, perhaps
```suggestion
	FIELD f_ugyhjati beamSegmentsToCheck Ljava/util/List;
```
?

File: mappings/net/minecraft/block/entity/BrewingStandBlockEntity.mapping [hunk 14]
```diff
@@ -5,9 +5,15 @@ CLASS net/minecraft/unmapped/C_sqxxaglx net/minecraft/block/entity/BrewingStandB
 	FIELD f_fxdchcgg fuel I
 	FIELD f_gylkmdei SIDE_SLOTS [I
 	FIELD f_hoaeyujo slotsEmptyLastTick [Z
+	FIELD f_hoibfolb DATA_VALUES I
+	FIELD f_ilynkgdg DATA_BREW_TIME I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/BrewingStandBlockEntity.mapping]:
```suggestion
	FIELD f_ilynkgdg BREW_TIME_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/BrewingStandBlockEntity.mapping [hunk 15]
```diff
@@ -5,9 +5,15 @@ CLASS net/minecraft/unmapped/C_sqxxaglx net/minecraft/block/entity/BrewingStandB
...
 	FIELD f_jlpksfdh inventory Lnet/minecraft/unmapped/C_rnrfftze;
 	FIELD f_odncodim TOP_SLOTS [I
+	FIELD f_rlzpnxen FUEL_SLOT I
+	FIELD f_tryldaqt INGREDIENT_SLOT I
+	FIELD f_ukqydstt FUEL_USES I
 	FIELD f_uxbdwavr BOTTOM_SLOTS [I
+	FIELD f_xgihhfkh DATA_FUEL I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/BrewingStandBlockEntity.mapping]:
```suggestion
	FIELD f_xgihhfkh FUEL_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/LecternBlockEntity.mapping [hunk 16]
```diff
@@ -1,7 +1,11 @@
 CLASS net/minecraft/unmapped/C_cctlzavr net/minecraft/block/entity/LecternBlockEntity
+	FIELD f_iecawanc DATA_VALUES I
 	FIELD f_ihinhmno currentPage I
 	FIELD f_oqozqfeg inventory Lnet/minecraft/unmapped/C_pjtstjoq;
 	FIELD f_oxehbgik book Lnet/minecraft/unmapped/C_sddaxwyk;
+	FIELD f_rvhjxmgp DATA_PAGE I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/LecternBlockEntity.mapping]:
```suggestion
	FIELD f_rvhjxmgp PAGE_DATA I
```
to more closely match spoken English

File: mappings/net/minecraft/block/entity/ShulkerBoxBlockEntity.mapping [hunk 17]
```diff
@@ -1,10 +1,17 @@
 CLASS net/minecraft/unmapped/C_gauehqwp net/minecraft/block/entity/ShulkerBoxBlockEntity
 	FIELD f_ainnhabf animationProgress F
 	FIELD f_axjnbhcn cachedColor Lnet/minecraft/unmapped/C_arllgqae;
+	FIELD f_ckaomydn CONTAINER_SIZE I
 	FIELD f_fmgzbbgv prevAnimationProgress F
+	FIELD f_gczipwyl ROWS I
 	FIELD f_gvwgoafu animationStage Lnet/minecraft/unmapped/C_gauehqwp$C_tdtslcqu;
+	FIELD f_kesedzww MAX_LID_ROTATION F
+	FIELD f_laidlwhf OPENING_TICK_LENGTH I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/block/entity/ShulkerBoxBlockEntity.mapping]: This sounds like it refers to the duration of a single tick, I assume it refers to a duration in (multiple) ticks, perhaps
```suggestion
	FIELD f_laidlwhf OPENING_TICKS I
```
or
```suggestion
	FIELD f_laidlwhf TICKS_TO_OPEN I
```
?

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Review by NoComment1105 — COMMENTED** [review]
Nothing to add that @supersaiyansubtlety hasn't added already

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]

**Pull request merged**